VideoAPI/VideoApi.DAL/Migrations/20210108162946_AddTansferRoomLabelToEvent.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace VideoApi.DAL.Migrations
+{
+    public partial class AddTansferRoomLabelToEvent : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "TransferredFromRoomLabel",
+                table: "Event",
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
+                name: "TransferredToRoomLabel",
+                table: "Event",
+                nullable: true);
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "TransferredFromRoomLabel",
+                table: "Event");
+
+            migrationBuilder.DropColumn(
+                name: "TransferredToRoomLabel",
+                table: "Event");
+        }
+    }
+}
```

Comment:
How about the rule: One migration per PR?

Reply:
These migrations are affecting different tables. The previous PR comment was in reference to multiple migrations to the same class in one PR